R/tm_missing_data.R

```diff
@@ -21,13 +21,13 @@
 #' ADSL <- teal.modules.general::rADSL
 #' ADRS <- teal.modules.general::rADRS
 #'
-#' app <- init(
+#' app <- teal::init(
 #'   data = cdisc_data(
 #'     teal.data::cdisc_dataset("ADSL", ADSL, code = "ADSL <- teal.modules.general::rADSL"),
 #'     teal.data::cdisc_dataset("ADRS", ADRS, code = "ADRS <- teal.modules.general::rADRS"),
 #'     check = TRUE
 #'   ),
-#'   modules = modules(
+#'   modules = teal::modules(
 #'     tm_missing_data(
```

Comment:
@kartikeyakirar what about teal.modules.general:: before tm_* functions from this package?